src/test/bin/quic_gtest.cpp

```diff
@@ -1116,6 +1116,7 @@ TEST_P(WithHandshakeArgs4, RandomLoss) {
             (uint8_t)GetParam().ServerStatelessRetry,
             0,  // ClientUsesOldVersion
             0,  // MultipleALPNs
+            0,  // GreaseQuicBitExtension
```

Comment:
How did this even build with a parameter number mismatch?

Reply:
It just didn't initialize the last value.